app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -27,18 +27,18 @@ def index
           zip_code: search_params['zip_code']
         )
         is_enhanced_ipp = resolved_authn_context_result.enhanced_ipp?
-        response = proofer.request_facilities(candidate, is_enhanced_ipp)
-        if response.length > 0
+        locations = proofer.request_facilities(candidate, is_enhanced_ipp)
```

Comment:
@rutvigupta-design I'm including screenshots for you to look at.

French
<img width="888" alt="Screenshot 2024-06-28 at 16 52 25" src="https://github.com/18F/identity-idp/assets/80347702/854bd93b-7610-4491-ae8e-2a41d88f8874">


Spanish
<img width="888" alt="Screenshot 2024-06-28 at 16 53 18" src="https://github.com/18F/identity-idp/assets/80347702/296938a9-0ba2-4ebd-b7f3-8a77d0b669d1">


Chinese
<img width="888" alt="Screenshot 2024-06-28 at 16 54 03" src="https://github.com/18F/identity-idp/assets/80347702/2c98bcca-3a52-46fd-956a-3e62b88b498e">

Reply:
The results LGTM but is there something strange going on in the H3 with all the languages? I don't think "lkn, lkn" in French, "kn, lkn" in Spanish, and "poj, ojpoj" should be there? 


<img width="383" alt="Screen Shot 2024-06-28 at 4 31 22 PM" src="https://github.com/18F/identity-idp/assets/51714306/0608a0e9-9de5-43bb-90a4-11c7644bbdb8">

<img width="395" alt="Screen Shot 2024-06-28 at 4 31 16 PM" src="https://github.com/18F/identity-idp/assets/51714306/9c4a70b0-375b-44d2-a578-ed45d2b389be">

<img width="364" alt="Screen Shot 2024-06-28 at 4 27 46 PM" src="https://github.com/18F/identity-idp/assets/51714306/4593b90e-34a4-4281-9e71-941ecb2db5d9">

Reply:
I think those are test values for the address/city fields in the search form

Reply:
Ahh, okay that makes sense, thanks for clarifying! This LGTM then!

Reply:
@rutvigupta-design Mitchell is correct, I typed in nonsense values for a test address.  Thanks for reviewing the PR! 🙏🏻